Tests/TSCBasicTests/ProcessTests.swift

```diff
@@ -92,6 +92,20 @@ class ProcessTests: XCTestCase {
             XCTFail()
         }
     }
+    
+    @available(macOS 12.0, *)
+    func testPopenAsyncAwait() async throws {
+        #if os(Windows)
+        let args = ["where.exe", "where"]
+        let answer = "C:\\Windows\\System32\\where.exe"
```

Comment:
You should not assume the path of `System` directory. Use `GetSystemDirectory` instead:
```suggestion
        var buffer = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
        guard GetSystemDirectoryW(&buffer, .init(buffer.count)) > 0 else {
            return XCTFail()
        }
        let answer = String(decodingCString: buffer, as: UTF16.self) + "\\where.exe"
```